Dockerfile

```diff
@@ -11,43 +11,37 @@ USER 0
 COPY package.json yarn.lock lerna.json tsconfig.json .prettierrc ./
 COPY apps ./apps
 COPY libs ./libs
-RUN chmod 0500 -R apps libs
 RUN apk add python3 make g++
 RUN sed -i s^https://registry.yarnpkg.com^$YARNREPO^g yarn.lock
 RUN yarn --frozen-lockfile --production --network-timeout 600000
 
 RUN yarn run build
-RUN chown -R node apps libs package.json
-RUN chmod 0400 package.json libs/interfaces/package.json tsconfig.json .prettierrc
-RUN chmod 0500 -R apps libs
 
 ### Production image
 
 FROM $BASE_CONTAINER as app
```

Comment:
following along with the snyk blog should we be setting a node_env variable?  i checked out setup env scripts and we're doing that in the dev env script (though setting the var to development) but not doing it at all in the docker env script.  i think the idea was that it would get passed through in the docker-compose (which is where we do have that var set to production), but then that means that it is not present when one tries using the image directly (e.x. running it locally or on kubernetes / non docker-compose system).  furthermore, should it be in both stages?  just this one?